app/templates/_pom.xml

```diff
@@ -172,6 +172,12 @@
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
             <version>${springfox.version}</version>
+            <exclusions>
```

Comment:
Why do you exclude MapStruct? This shouldn't be in this commit, it has nothing to do with i18n